src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -5098,8 +5097,8 @@ public static boolean isNotEmpty(final boolean[] array) {
              */
             final Class<?> type2 = array2.getClass().getComponentType();
             if (!type1.isAssignableFrom(type2)) {
-                throw new IllegalArgumentException("Cannot store " + type2.getName() + " in an array of "
-                        + type1.getName(), ase);
+                throw IllegalArgumentExceptions.format(ase, "Cannot store %s in an array of %s", type2.getName(),
+                        type1.getName(), ase);
```

Comment:
Why is the ase parameter listed twice? Looks wrong

Reply:
I will remove the last `ase`. The rest is correct, it is not the same, one param is from `type2` and the next for `type1`.